var/spack/repos/builtin/packages/watch/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Watch(AutotoolsPackage):
+    """Executes a program periodically, showing output fullscreen."""
+
+    homepage = "https://gitlab.com/procps-ng/procps"
```

Comment:
Can you leave a comment on why you did this (`procps` can't build on `darwin`) and a few references?